name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/dataseries/DataSeriesChartLegend.java

```diff
@@ -242,6 +245,19 @@ public void run()
                 });
                 actionShowArea.setChecked(series.isShowArea());
                 manager.add(actionShowArea);
+                
+                MenuManager lineWidth = new MenuManager(Messages.ChartSeriesPickerLineWidth);
+                IntStream.range(1, 3).forEach(i ->{
+                    Action action = new SimpleAction(i + " px", a -> { //$NON-NLS-1$
+                        series.setLineWidth(i);
+                        configurator.fireUpdate();
+                    });
+                    action.setChecked(i==series.getLineWidth());
```

Comment:
`action.setChecked(i == series.getLineWidth());` :wink:

Reply:
Thanks for the hints. I just updated all your points. Somehow the ResourceBundle Editor has also automatically adjusted the encoding of one other property. But that should be OK.